packages/sample/src/main.ts

```diff
@@ -6,6 +6,9 @@ const program = new Command();
 
 program.version('1.0.0')
     .option('-l --list', 'List connected devices')
+    .option('-c --count <counts>', 'Set encoder count.')
+    .option('-m --motor <index>', 'Set motor index. Must also include -p or -c')
+    .option('-p --power <power>', 'Set motor power')
```

Comment:
`-c` and especially `-p` should only work if `-m` is specified. You don't know what the motors in question are hooked up to, and you don't want the wrong thing to start spinning if they forget to specify which motor. Also, IMO the `-m` description should reference the long forms of options, not the short forms.